templates/repo/branch/list.tmpl

```diff
@@ -13,10 +13,14 @@
 				<tbody>
 					<tr>
 						<td>
-							{{if .IsProtected}}
-								<i class="octicon octicon-shield"></i>
+							{{range $branch := .Branches}}
```

Comment:
Imho if you define variable .IsProtected still refers to parent object, no?

Reply:
Oh, missed that, you should use `{{range .Branches}}`

Reply:
but we shut fix that upstream to!!!